.github/ISSUE_TEMPLATE/bug_report.yml

```diff
@@ -0,0 +1,79 @@
+name: Bug Report
+about: Report an Electron bug
+title: "[Bug]: "
+body:
+- type: checkboxes
+  attributes:
+    label: Preflight Checklist
+    description: Please ensure you've completed all of the following.
+    options:
+      - label: I have read the [Contributing Guidelines](https://github.com/electron/electron/blob/master/CONTRIBUTING.md) for this project.
+        required: true
+      - label: I agree to follow the [Code of Conduct](https://github.com/electron/electron/blob/master/CODE_OF_CONDUCT.md) that this project adheres to.
+        required: true
+      - label: I have searched the [issue tracker](https://www.github.com/electron/electron/issues) for a feature request that matches the one I want to file, without success.
+        required: true
+- type: input
+  attributes:
+    label: Electron Version
+    description: What version of Electron are you using?
+    placeholder: 12.0.0
+  validations:
+    required: true
+- type: dropdown
+  attributes:
+    label: What platform are you using?
```

Comment:
We should split this into platform and arch (two dropdowns)

Platform: macOS/Windows/Linux
Platform version: text input
Arch: x64/ia32/arm64(including apple silicon)

Reply:
@MarshallOfSound how do you envision also allowing for specifying Big Sur, Mojave, Catalina, etc?